src/Common/DataFileUtils.cs

```diff
@@ -14,29 +20,54 @@ public static void EnsureOutputDirectory(string outputFile)
         }
     }
 
-    // Ensure text written into tab-separated format is collapsed onto a single line
-    // without any false-positive matches for tabs, and ensuring quotes don't lead
-    // to the tab separator characters being unrecognized.
-    public static string SanitizeText(string text) => text
+    /// <summary>
+    /// Sanitizes the specified text by replacing certain characters to ensure it is collapsed onto a single line.
+    /// </summary>
+    /// <param name="text">The text to sanitize.</param>
+    /// <returns>The sanitized text.</returns>
+    public static string SanitizeText(string text)
+        => text
         .Replace('\r', ' ')
         .Replace('\n', ' ')
         .Replace('\t', ' ')
-        .Replace('"', '`');
+        .Replace('"', '`')
+        .Trim();
```

Comment:
@jeffhandley this shouldn't be a problem, should it?

Reply:
Adding `.Trim()`? I can't think of any problems there.